src/outbreak_data/outbreak_data.py

```diff
@@ -162,61 +164,46 @@ def prevalence_by_location(location, ndays=180, nday_threshold=10, other_thresho
 
 
 
-def lineage_mutations(pango_lin, mutations=None, freq=0.8, server=server, auth=None):
+def lineage_mutations(pango_lin, mutations=None, freq=0.8, server=server, auth=None):  ###
     """Retrieves data from all mutations in a specified lineage above a frequency threshold.
-       - Mutiple queries for lineages and mutations can be separated by ","
        - Use 'OR' in a string to return overlapping mutations in multiple lineages: 'BA.2 OR BA.1'
-       - param mutation is only useful for one lineage + mutation1 + mutation2 .... combinations
 
           Arguments:
-             :param pango_lin: A string or list; loads data for all mutations in a specified PANGO lineage 
-             :param mutation: A string or list; loads data of mutations for sequences classified as a specified PANGO lineage with mutation
+             :param pango_lin: A string; loads data for all mutations in a specified PANGO lineage 
+             :param mutations: A string; loads mutation data for the specified sequence under the specified PANGO lineage 
              :param freq: A number between 0 and 1 specifying the frequency threshold above which to return mutations (default = 0.8)
              :return: A pandas dataframe"""
 
-    # Turns any string input into list format: most universal
-    if isinstance(pango_lin, str) is True:
-      if 'OR' in pango_lin:
-          lineages = pango_lin.split('OR')
-          lineages = "OR".join(lineages)
-      else:
-          lineages = pango_lin.replace(" ", "")
-          lineages = lineages.split(',')
-          lineages = ",".join(lineages)
-    elif isinstance(pango_lin, list):
-         lineages = ",".join(pango_lin)
-                  
+    # Use strings, no reason to use list format anymore
+   
+        
+    if isinstance(pango_lin, str):
+        if '*' in pango_lin:
+            query = '&' + f'q=pangolin_lineage_crumbs:*;{pango_lin};*'
+        else:
+            query = pango_lin
+            if 'OR' in pango_lin:
+              lineages = pango_lin.split('OR')
+              query = "OR".join(lineages)
+              
     if mutations:
-        if isinstance(mutations, str) is True:
-            mutations = mutations.replace(" ", "")
-            mutations = list(mutations.split(","))   # deals with string format for mutations
-
-        if isinstance(mutations, list) is True:
-            mutations = " AND ".join(mutations)
-        mutations = " AND " + mutations
-        lineages = '' + lineages + '' + mutations # fixed function
+        query = query + '&' + f'mutations={mutations}'
         
-        raw_data = get_outbreak_data('genomics/lineage-mutations', f'pangolin_lineage={lineages}', collect_all=False)
-        key_list = raw_data['results']
-        if len(key_list) == 0:
-            raise TypeError('No matches for query found')
+    raw_data = get_outbreak_data('genomics/lineage-mutations', f'lineages={query}', collect_all=False)
```

Comment:
This is because of an API bug, not your code, but right now calling `outbreak_data.lineage_mutations('AY.4*', mutations='S:E484K')` results in an error. We can fix it for now by changing the query from this `https://api.outbreak.info/genomics/lineage-mutations?lineages=&q=pangolin_lineage_crumbs:*;AY.4*;*&mutations=S:E484K`, to this `https://api.outbreak.info/genomics/lineage-mutations?lineages=None&mutations=S:E484K&q=pangolin_lineage_crumbs:*;AY.4;*` -- `q=` needs to be the last parameter, and we need to set `lineages=None` for now